src/function/vector_cast_operations.cpp

```diff
@@ -65,5 +85,14 @@ CastToStringVectorOperation::getDefinitions() {
     return result;
 }
 
+std::vector<std::unique_ptr<VectorOperationDefinition>>
+CastToDoubleVectorOperation::getDefinitions() {
+    std::vector<std::unique_ptr<VectorOperationDefinition>> result;
+    result.push_back(make_unique<VectorOperationDefinition>(CAST_TO_DOUBLE_FUNC_NAME,
+        std::vector<DataTypeID>{INT64}, DOUBLE,
+        UnaryExecFunction<int64_t, double_t, operation::CastToDouble>));
```

Comment:
reuse bindExec to get your exec function

Reply:
done